src/gentropy/dataset/variant_index.py

```diff
@@ -22,11 +22,48 @@
     from pyspark.sql.types import StructType
 
 
-
 @dataclass
 class VariantIndex(Dataset):
     """Dataset for representing variants and methods applied on them."""
 
+    CONSEQUENCE_TO_PATHOGENICITY_SCORE = [
```

Comment:
No, the mapping attribute is not going to be dropped. What is going to be changed is how the VEP features are built. We have this method `variant_index.get_most_severe_gene_consequence`that maps each consequence with the score and gets the highest. Because this is already being done in the variant index, the function is simpler. Let me change it now.

Reply:
Ok